source/databases-collections.txt

```diff
@@ -119,31 +119,39 @@ initial collection.
 
 To create a new database:
 
-1. Hover over the connection for the deployment where you want your
+#. Hover over the connection for the deployment where you want your
```

Comment:
Actually not sure why this is documented twice, the procedure is also here:
https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-vscode/docsworker-xlarge/DOCSP-28311/playground-databases/#create-a-database-and-collection.

Maybe it has value to breakout the task into database and collection creation.

Reply:
Yeah, I think a refactor would be beneficial here to align this content with our IA templates. It's not particularly useful to have this information documented twice.

One idea is that we could tailor the "Navigate Your Data" section to be about _existing_ data, and talk about creating new databases and collections exclusively in the context of playgrounds. Not sure if this is the best approach, but might be one way to cut down on duplication.

I'd consider these updates out of scope for this specific ticket.